src/components/home/Home.js

```diff
@@ -36,45 +46,69 @@ const useStyles = makeStyles((theme) => ({
   typographyMargin: {
     margin: theme.spacing(1.5, 0),
   },
+  searchOptions: {
+    marginTop: theme.spacing(-10),
+    marginLeft: theme.spacing(1),
+  },
 }));
 
 function Home() {
   const { t } = useTranslation();
   const classes = useStyles();
   const [searchResults, setSearchResults] = useState(null);
+  const [typing, setTyping] = useState(null);
+  const [range, setRange] = useState('all');
```

Comment:
Use enum (again you will need to create it again)..

Reply:
It says only typescript file could have enum, so I use a dict instead